src/srb2kart/serverinfo/serverinfo.go

```diff
@@ -0,0 +1,297 @@
+package serverinfo
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"net"
+)
+
+var getServerInfoPacket = [...]byte{
+	0x58, 0x46, 0x23, 0x01,
+	0x00,
+	0x00,
+	0x0C,
+	0x00,
+
+	0x01,
+	0x1f, 0x02, 0x00, 0x00,
+}
+
+type packetType uint8
+
+const (
+	packetTypeServerInfo packetType = 0x0D
+	packetTypePlayerInfo packetType = 0x0E
+)
+
+type KartPacket interface{
+	GetPacketType() packetType
+}
+
+type KartPacketHeader struct {

```

Comment:
No need for public, it's only applicable in this package

Reply:
Should the `Header` field in the public packet structs remain public if this is made private?

Reply:
I wouldn't think so. The header is only important for the networking thing. The body is for the user of the package.